libs/execution_context/ebpf_program.c

```diff
@@ -249,9 +249,6 @@ ebpf_program_create(ebpf_program_t** program)
         goto Done;
     }
 
-    ebpf_list_initialize(&local_program->links);
-    ebpf_lock_create(&local_program->links_lock);
```

Comment:
This change looks like it's reverting PR #395.  Is that intentional?

Reply:
No, not intentional